kibana/install.sls

```diff
@@ -15,4 +15,7 @@ include:
 kibana-pkg:
   pkg.installed:
     - name: {{ kibana.pkg }}
+{% if kibana.repoVersion.startswith('5.') %}
```

Comment:
You should stick to the style of the rest of the file unless you have a reason. i.e. Your `{&` should be `{%-`.

Reply:
@blbradley Hey there, I really don't understand the `-` operator in jinja2.

Documentation says 

> You can also strip whitespace in templates by hand. If you add a minus sign (-) to the start or end of a block (e.g. a For tag), a comment, or a variable expression, the whitespaces before or after that block will be removed:

To be honest, I am not a programmer, I am somewhat of a beginner in python and in java, so the concept of "whitespace" is relatively new to me. I'll edit all files and add the `-` operator.

Reply:
Hi! 😄 

For SaltStack in Jinja, that operator is commonly used to control if an newline appears or not for a conditional statement.

Does that clear up your knowledge? Feel free to follow up with questions.

Reply:
@blbradley ah so without `-` operator the final rendered file should look like

```
kibana-pkg:
     pkg.installed:
         - name: kibana

         - version: 5.1.1
```

and with the `-` operator

```
kibana-pkg:
     pkg.installed:
         - name: kibana
         - version: 5.1.1
```

Reply:
Correct!